.github/workflows/pull-request.yml

```diff
@@ -106,11 +106,13 @@ jobs:
           path: node_modules
 
       - name: Setup Node
+        if: ${{ github.base_ref == 'main' }}
```

Comment:
Could move this up to the job level so the whole job is skipped if the pull request isn't targeting `main`?

Reply:
Ah... we could change the branch protection rules so that they only apply for pull requests targeting main I think? 

Looks like it should be possible with the new (more complicated) "Rulesets"?

<img width="815" alt="Screenshot 2025-03-13 at 09 03 05" src="https://github.com/user-attachments/assets/9f8dcea0-59b9-4f88-a6c7-cc9b747a0585" />

Reply:
@colinrotherham I don’t have admin access to this repo, but @anandamaryon1 does - I asked him to tweak the settings after we split out the tests into parallel jobs last time.